src/sentry/incidents/logic.py

```diff
@@ -471,6 +480,17 @@ class AlertRuleNameAlreadyUsedError(Exception):
 }
 
 
+def get_alert_resolution(time_window: int, organization) -> int:
+    resolution = DEFAULT_ALERT_RULE_RESOLUTION
+
+    if features.has("organizations:metric-alert-load-shedding", organization=organization):
+        resolution = DEFAULT_ALERT_RULE_LOAD_SHEDDING_RESOLUTIONS.get(
+            time_window, DEFAULT_ALERT_RULE_RESOLUTION
+        )
```

Comment:
It might be wise to structure this as ranges, so 1-29 is 1, 30-59 is 2, 60-239 is 3, 240-999 is 5, 1000-1440 is 15.

Either that or we should more strictly enforce these buckets at the api level

Reply:
👍 i like the idea of using the ranges based on the dicts.

Yeah, I agree about the API level... it'd be nice if the API enforced the select values in the UI. I'm hoping to get a bit more time in the next quarter to work on some things like that around alerts, I'll add it to my list!